utils/composer-packages/src/Rector/AddPackageVersionRector.php

```diff
@@ -56,29 +56,27 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
-        if (! property_exists($node, 'stmts')) {
-            return null;
-        }
+        /** @var Closure $closure */
+        $closure = $node;
 
         /** @var Expression $stmt */
-        foreach ($node->stmts as $stmt) {
+        foreach ($closure->stmts as $stmt) {
             if (! $stmt->expr instanceof Assign) {
                 continue;
             }
 
-            if (! $this->isName($stmt->expr->var, 'composerExtensions')) {
-                continue;
-            }
+            $assign = $stmt->expr;
 
-            if (! $stmt->expr->expr instanceof Array_) {
+            if (! $this->isName($assign->var, 'composerExtensions')) {
                 continue;
             }
 
-            if (! property_exists($stmt->expr->expr, 'items')) {
+            if (! $assign->expr instanceof Array_) {
                 continue;
             }
 
-            $stmt->expr->expr->items[] = new ArrayItem(
```

Comment:
During testing, I realized that PhpStorm cannot handle multinesting instanceof.

E.g.

```php
if ($node->expr->expr instanceof String_) {
     $node->expr->expr // mixed?
}
```

But can remember one property fetch:

```php
if ($node->expr instanceof String_) {
     $node->expr // String_ of course
}
```

I used this trick here, to make PhpStorm always remember the type :+1: